amz_review_scraper/amzscraper.py

```diff
@@ -70,12 +72,12 @@ def scrape(soup, asin):
     for divs in soup.findAll("div", attrs={"data-hook": "review-collapsed"}):
         long_review = divs.text.strip()
         try:
-            scraped_review = review.Review(
-                review=long_review, asin=asin, owner=scraped_item
-            )
-            review_exists = scraped_review.check()
-            if review_exists == None:
-                scraped_review.save()
+            review_exists = review.get_results(asin=asin, review=long_review)
+            if review_exists is None:
+                scraped_review = review.Review(
+                    review=long_review, asin=asin, owner=scraped_item
+                )
+                review.save(scraped_review)
             else:
```

Comment:
its good to add some message when a path that does nothing is hit. It could be hard to debug later, if you didn't know a branch of a conditional ran.